java/com/google/re2j/JavaCategoryTables.java

```diff
@@ -0,0 +1,815 @@
+
+package com.google.re2j;
+
+// AUTOGENERATED by MakeJavaCategories.java - do not modify
```

Comment:
You shouldn't check in autogenerated files.  They should be generated as part of the build process.